Update methods.mdx (#86)
devrev/fern-api-docs: fern/docs/pages/plug-sdk/methods.mdx

<Callout intent="note">
When React is in Strict mode, you may receive a warning about `window.plugSDK.init()` being called multiple times. These warnings do not affect the widget's installation or functionality.
</Callout>

## Toggle widget theme

The toggle theme method lets you dynamically adjust the PLuG SDK’s theme, even after the widget has been initialized. This feature allows for real-time theme changes based on your users' preferences.

The method accepts the input options: `light` and `dark`.

```jsx
window.plugSDK.toggleTheme('light | dark');
```

The parameter is optional; calling `toggleTheme()` will switch between the current themes. Specifying a theme as a parameter allows you to toggle directly to that specific theme.

<Callout intent="note">
The theme specified in this method takes precedence over the theme configured in the PLuG settings in your DevRev app.
</Callout>

## Toggle widget

The `toggleWidget` method allows you to control the visibility of the PLuG chat widget.

> Passing input = true opens the chat widget.

Reviewer: ```suggestion
Passing `true` opens the chat widget.
```

> Passing input = false closes the chat widget.

Reviewer: ```suggestion
Passing `false` closes the chat widget.
```

```jsx
window.plugSDK.toggleWidget(true);
```

## Take action from PLuG chat events

> This method allows you to listen for events from the PLuG widget. Below are the available `PAYLOAD_TYPES`.

Reviewer: It's TYPE, not TYPES.
```suggestion
This method allows you to listen for events from the PLuG widget. Below are the available values for `PAYLOAD_TYPE`.
```
